templates/show_test_table.inc.php

```diff
@@ -164,6 +164,21 @@
     <?php echo T_('This tests whether Ampache can manage large files (> 2GB). This is not strictly necessary, but may result in a better experience. This generally requires 64-bit operating system.'); ?>
     </td>
 </tr>
+<tr>
+    <td valign="top"><?php echo T_('PHP mbstring.func_overload'); ?></td>
+    <td valign="top">
+    <?php echo debug_result(check_mbstring_func_overload()); ?>
+    </td>
+    <td>
+    <?php printf(T_('This tests whether PHP %s is set as it may break the ID3 tag support.'), '<a href="http://php.net/manual/en/mbstring.overload.php">mbstring.func_overload</a>'); ?>
+    <?php
+    if (!check_mbstring_func_overload()) {
```

Comment:
Don't do an additional check here, better to have only one unique sentence describing the test (why would we add specific process on this check and not on the other?).